openhtf/io/test_record.py

```diff
@@ -29,6 +29,7 @@
 from openhtf.util import logs
 
 _LOG = logging.getLogger(__name__)
+CAPTURE_SOURCE_CODE = True  # Override before creating an openhtf.Test()
```

Comment:
hm, i think it is a bit kludgey to have tunable knobs like this (that require poking between imports), but there is precedent for it (with logging, for example, and other modules like GUI tools where you have to configure a backend before importing other bits).

i think it's a pretty close tradeoff in this case, so if you'd prefer to keep it this way, i'm okay with it.  it does need to be clearly documented how to do this, though, and what the constraints are on when that attribute needs to be set.  with updated docs, LGTM.